livre/eq/zeq/communicator.cpp

```diff
@@ -259,7 +259,7 @@ class Communicator::Impl
         // Receiving all queued events from all receivers without blocking.
         for( auto subscriber : subscribers )
             while( subscriber->receive( 0 ))
-                /*nop*/ ;
+                _config.sendEvent( REDRAW );
```

Comment:
So, long term integration tests, short term `me` with warnings ( and refactoring ).

The redraw being send from `any` event may not be the best idea. Maybe we send it with some specific events ? ( Or for all events, the redraw is needed ? )